modules/database_module.rb

```diff
@@ -0,0 +1,18 @@
+module Database
+  FOLDER_PATH = './database/'.freeze
+  FILE_PATH = 'test'.freeze
+  FORMAT_PATH = '.yml'.freeze
+  PATH = FOLDER_PATH + FILE_PATH + FORMAT_PATH
+
+  def input_to_db(entity)
+    entities = add(entity)
```

Comment:
here you use `add` method of class, YOU SHOULDN'T use method of includable class